include/openenclave/internal/utils.h

```diff
@@ -49,6 +49,15 @@ OE_INLINE bool oe_is_ptrsize_multiple(size_t n)
     return (d >= 1 && r == 0);
 }
 
+OE_INLINE bool oe_is_size_multiple_alignment(size_t size, size_t alignment)
+{
+    if (alignment == 0)
+        return false;
+    size_t d = size / alignment;
```

Comment:
Is `d` (quotient) needed here? Can this be accomplished by just using the remainder?

Reply:
It's needed to assure that size>0